modules/doc/content/help/inl/hpc_remote.md

```diff
@@ -55,23 +55,16 @@ domain name in your command above because of the "Host" setting in your SSH conf
 
 ## SOCKS Proxy id=socks-proxy
 
-Adjust the SOCKS proxy settings for your +least+ favorite web browser to reflect the following
-settings:
+In order to access common HPC resources (hpcgitlab.hpc.inl.gov, hpcweb.inl.gov, etc) within a web browser, you must tunnel traffic to said resources using a SOCKS proxy. This is achieved by using a PAC (Proxy-Auto Configuration) file, which we have configured as:
```

Comment:
```suggestion
To access common HPC resources (hpcgitlab.hpc.inl.gov, hpcweb.inl.gov, etc) within a web browser, traffic must be routed through a SOCKS proxy. This can be achieved by using a PAC (Proxy-Auto Configuration) file:

```